problem-28.c

```diff
@@ -0,0 +1,39 @@
+#include <stdio.h>
+// Uses structures to show names, roll numbers and percentages of 5 students
+
+typedef struct
+{
+    char *name;
+    int roll_number;
+    float percentage;
+} Student;

```

Comment:
Any particular reason of defining struct like this and not as following?

```c
struct myStructure {
  int myNum;
  char myLetter;
};
```
I think its better to have a struct name at the start, which specially helps if the struct is long.

Reply:
Because defining it using `typedef` and name after definition we don't have to write `struct Student` every time we want to use it. Instead we can use `Student` only